wagtail/wagtailadmin/static_src/wagtailadmin/js/page-editor.js

```diff
@@ -34,11 +39,9 @@ function makeRichTextEditable(id) {
 
     var closestObj = input.closest('.object');
 
-    richText.hallo({
-        toolbar: 'halloToolbarFixed',
-        toolbarCssClass: (closestObj.hasClass('full')) ? 'full' : (closestObj.hasClass('stream-field')) ? 'stream-field' : '',
-        plugins: halloPlugins
-    }).bind('hallomodified', function(event, data) {
+    halloConfig.toolbarCssClass = (closestObj.hasClass('full')) ? 'full' : (closestObj.hasClass('stream-field')) ? 'stream-field' : '';
```

Comment:
If we want to give ability to override `toolbarCssClass`, line `$.extend(halloConfig, editorConfig);` should be after this line.